src/doc/trpl/structs.md

```diff
@@ -157,13 +136,9 @@ struct Point {
 }
 ```
 
-Now, we have actual names, rather than positions. Good names are important,
-and with a struct, we have actual names.
+今、私たちはフィールドの位置ではなくフィールドの名前を持っています。良い名前は重要で、構造体を使うということは、私たちは名前を持っているということです。(訳注:タプルはフィールドの並びによって区別され、構造体は名前によって区別されます。これはタプルと構造体の最たる違いであり、構造体を持つことは名前付けられたデータの集まりを持つことに等しいため、構造体における名前付けは重要です)
 
-There _is_ one case when a tuple struct is very useful, though, and that’s a
-tuple struct with only one element. We call this the ‘newtype’ pattern, because
-it allows you to create a new type, distinct from that of its contained value
-and expressing its own semantic meaning:
+タプル構造体が非常に便利な場合も_あります_が、1要素で使う場合だけです。タプル構造体の中に入っている値や式と、それ自身のセマンティクスを明確に区別できるような新しい型を作成できることから、私たちはこれを'newtype'パターンと呼んでいます。
```

Comment:
セマンティクスとカタカナ表記にされていますが、原文ではsemantic meaningとされているので"semantic"な「意味」であることを主張したいように解釈できると思うのですが如何でしょうか。

また、

>Now, we have actual names, rather than positions. Good names are important, and with a struct, we have actual names.

の訳についてですが、訳注も含めこの訳で問題無いかと思います。